### PR TITLE
Node 14 recomendation

### PR DESCRIPTION
**Pull request by artemkovalyov**
## What Has Changed?

Explain what you have changed.

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)

### CODE REVIEW

File: docs/js/guides/upgrade-to-version-2.mdx [hunk 1]
```diff
@@ -32,6 +32,12 @@ If you run node version 12 or greater, no changes are necessary.
 If you run an older node version, please update to a newer version.
 You can find a list of breaking changes in the news section of the [node.js website](https://nodejs.org/en/blog/) (e.g. ["The Difference Between Node.js 10 and 12"](https://nodejs.org/en/blog/uncategorized/10-lts-to-12-lts/))
 
+:::tip 
+
+We recommend to switch to Node 16 when using the SAP Cloud SDK for JavaScript 2.0.
```
Comment (jjtang1985) [inline docs/js/guides/upgrade-to-version-2.mdx:37]: We don't have node 16 test in our pipeline, unfortunately. For the time being, maybe mentioning >= node 14

Comment (artemkovalyov) [inline docs/js/guides/upgrade-to-version-2.mdx:37]:
```suggestion
We recommend switching to Node 14 or above when using the SAP Cloud SDK for JavaScript 2.0.
```

Comment (artemkovalyov) [reply docs/js/guides/upgrade-to-version-2.mdx]: I changed it, take another look.

**Review by jjtang1985 — APPROVED** [review]
LGTM.
Maybe you can update the title of the PR.

**Pull request merged**